src/metadata/ColumnMetadata.ts

```diff
@@ -532,7 +532,7 @@ export class ColumnMetadata {
      * If column is in embedded (or recursive embedded) it extracts its value from there.
      */
      getEntityValue(entity: ObjectLiteral, transform: boolean = false): any|undefined {
-        // if (entity === undefined || entity === null) return undefined; // uncomment if needed
```

Comment:
so, why its needed?)

Reply:
Because in some case when you actually insert multiple rows which cause a conflict `entity` is not defined, so therefore line 595 will break `value = entity[this.propertyName];`

Database with columns c1, c2, c3 with a unique index on c1 and the following data
```
| A (c1) | Foo (c2) | Bar (c3) |
```
If i try to insert the following
```
| A | Foo2 | Bar2 |
| A | Foo3 | Bar3 |
```
I just want the database to be  `| A | Foo3 | Bar3 |` i guess this has something to do with the fact that you are actually trying to insert N rows but returning N-1

Reply:
mm not sure I got it, but okay, lets merge it